src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import converter.DataConverterImpl;
+import handler.BalanceOperationHandler;
+import handler.OperationHandler;
+import handler.PurchaseOperationHandler;
+import handler.ReturnOperationHandler;
+import handler.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import model.FruitTransaction;
+import read.write.file.FileReaderImpl;
+import read.write.file.FileWriterImpl;
+import report.ReportGeneratorImpl;
+import service.ShopServiceImpl;
+import strategy.OperationStrategyImpl;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    // HINT: In the `public static void main(String[] args)`
+    // it is better to create instances of your classes,
+    // and call their methods, but do not write any business logic in the `main` method!
+    public static void main(String[] args) {
```

Comment:
Make sure all services are independent and called in the right order in the `main()` method. The instantiation and usage of `FruitShopManager` indicate that it might be handling too many responsibilities. Consider refactoring to ensure that services like file reading, processing, and writing are called in sequence in the `main()` method, rather than being encapsulated in a manager class.